dbus_next/aio/message_bus.py

```diff
@@ -28,14 +28,19 @@ class MessageBus(BaseMessageBus):
     :type bus_type: :class:`BusType <dbus_next.BusType>`
     :param bus_address: A specific bus address to connect to. Should not be
         used under normal circumstances.
+    :param expanded_intr: An option to select expanded interface tree in introspection or just
+        exported interfaces.
+    :type expanded_intr: bool
```

Comment:
What's the use case for this?

Reply:
This is just an effect of some experimentation. Some dbus libraries return 'expanded' while others 'collapsed' Introspection. I thought this could be, at no great cost, an option left to developer to choose. 
I don't have very strong feelings toward this however.

Reply:
Complexity of the api is a big concern for me because it's already way too complicated. So maybe we should put this off until someone needs it.

Reply:
Option removed